doc/Metrics.md

```diff
@@ -5,15 +5,17 @@ EdgeHub is intrumented to collect the following metrics:
 | MetricName        | Description           | Unit  |
 | ------------- |:-------------:|:-----:|
 | EdgeHubToCloudMessageLatencyMs | Time taken by EdgeHub to send a message to the cloud | Milliseconds |
-| EdgeHubToCloudMessageSentCount | Number of messages sent by EdgeHub to the cloud | Count per reporting interval* |
+| EdgeHubToCloudMessageSentCount | Number of messages sent by EdgeHub to the cloud | Count resets each reporting interval* |
+| EdgeHubConnectedClientGauge | Number of clients/devices currently connected to EdgeHub | Count |
 | EndpointMessageStoredLatencyMs | Time taken by EdgeHub to acknowledge receipt of a message | Milliseconds |
-| EndpointMessageStoredCount | Number of messages stored by EdgeHub | Count per reporting interval* | 
+| EndpointMessageStoredCount | Total number of messages stored by EdgeHub | Last recorded total | 
+| EndpointMessageDrainedCount | Total number of messages sent to cloud by EdgeHub | Last recorded total |
```

Comment:
Should we call this `EdgeHubToCloudMessageTotalCount` or something?

Reply:
Hmm. Actually its not just number of messages sent to the cloud. Its tagged per endpoint. I have to change the description here.